src/views/Question.vue

```diff
@@ -60,13 +63,30 @@ import QuizMultiChoice from '@/components/QuizMultiChoice.vue'
     let categoryAnswers = category.questions[questionID-1].answers;
 
     return {
-      title: category.title,
+      title: category.title.toUpperCase(),
```

Comment:
Should use CSS to uppercase instead of JS, because it's a visual representation of the text. Can use `text-transform: uppercase`.